projects/scion/microfrontend-platform/src/lib/host/manifest-registry/ɵmanifest-registry.ts

```diff
@@ -99,15 +99,23 @@ export class ɵManifestRegistry implements ManifestRegistry, PreDestroy {
    */
   private hasIntentionForCapability(appSymbolicName: string, capability: Capability): boolean {
     const filter: ManifestObjectFilter = {appSymbolicName, type: capability.type, qualifier: capability.qualifier};
```

Comment:
Same here. Why are we passing the qualifier in the filter?
Or better, I think, here, we do not need to pass a qualifier predicate at all, simply pass the qualifier in the filter.